tests/test_leanplum_export.py

```diff
@@ -93,7 +98,7 @@ def set_contents(filename):
 
         mock_bucket, mock_client, mock_blob = Mock(), Mock(), Mock()
 
-        mock_client.list_blobs.side_effect = [[]]
+        mock_client.list_blobs.return_value = BlobList([])
```

Comment:
Alternatively, use `Mock()` and then set the `return_value` for `pages`

Reply:
Ah `PropertyMock` was what I was forgetting about.